sydent/http/servlets/store_invite_servlet.py

```diff
@@ -178,29 +178,31 @@ def _redact(self, s, characters_to_reveal):
         :type s: unicode
 
         :param characters_to_reveal: How many characters of the string to leave before
-            the '...'
+            the ellipsis
         :type characters_to_reveal: int
 
         :return: The redacted string.
         :rtype: unicode
         """
+        ellipsis_character = u"…"
```

Comment:
This could probably just be inlined in each place it is used? Unless there's a plan to make this configurable. If you want to keep it, I might call this `obfuscation_string` or something like that.

Reply:
Not really a plan, no. I think I did it out of habit, but it indeed doesn't bring much value.